### PR TITLE
update free-Courses-en

### PR DESCRIPTION
**Pull request by Ttecs**
updated react native playlist

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
:writing_hand: It's a good practice to fill Pull Request fields. It helps a bit we can understand your contribution better :hugs:.

**Comment by Ttecs** [comment]
## What does this PR do?
Add resource 

## For resources : React Native Beginer tutorial Series aded
### Description :youtube tutorial series for React Native

### Why is this valuable (or not)? it is latest and good for beginers

### How do we know it's really free?it  in you tube


## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -493,6 +493,7 @@
 * [CS50's Mobile App Development with React Native](https://www.edx.org/course/cs50s-mobile-app-development-with-react-native) - edX
 * [Introduction to React Native](https://fullstackopen.com/en/part10/introduction_to_react_native) - Full Stack Open
 * [React Native for Beginners](https://www.youtube.com/playlist?list=PL4cUxeGkcC9ixPU-QkScoRBVxtPPzVjrQ) - The Net Ninja
+* [React Native Tutorial (2021)](https://youtube.com/playlist?list=PL8kfZyp--gEXs4YsSLtB3KqDtdOFHMjWZ) - Programming with Mash
```
Comment (davorpa) [inline courses/free-courses-en.md:496]:
```suggestion
* [React Native Tutorial (2021)](https://youtube.com/playlist?list=PL8kfZyp--gEXs4YsSLtB3KqDtdOFHMjWZ) - Programming with Mash, MAhdi SHarifimehr (YouTube)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to make some changes. When done, request for a review again using GitHub :arrows_clockwise: button.

**Review by davorpa — APPROVED** [review]
Thanks :hugs:, have a warm welcome to `free-programming-books` :heart: :books:

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**